sympy/polys/subresultants_qq_zz.py

```diff
@@ -53,6 +53,15 @@
 called the resultant of f, g and serves as a criterion of whether
 the two polynomials have common roots or not.
 
+In sympy the resultant is computed with the function
+resultant(f, g, x). This function does _not_ evaluate the
+determinant of sylvester(f, g, x, 1); instead, it returns
+the last member of the subresultant prs of f, g!
+
+Caveat: If Df = degree(f, x) and Dg = degree(g, x), then:
+
+resultant(f, g, x) = (-1)**(Df*Dg) * resultant(g, f, x).
```

Comment:
I see, I have to change the wording a bit to account for the change of  
the order of the functions. If the sign is added, then we would have  
to say
something like:


>> + ...; instead, it returns +the last member of the subresultant prs  
>> of f, g, +multiplied by a factor of (-1) as explained in the caveat  
>> below.

Do you want me to change it now?


Quoting Kalevi Suominen <notifications@github.com>:

>> @@ -53,6 +53,15 @@
>>  called the resultant of f, g and serves as a criterion of whether
>>  the two polynomials have common roots or not.
>>
>> +In sympy the resultant is computed with the function
>> +resultant(f, g, x). This function does _not_ evaluate the
>> +determinant of sylvester(f, g, x, 1); instead, it returns
>> +the last member of the subresultant prs of f, g!
>> +
>> +Caveat: If Df = degree(f, x) and Dg = degree(g, x), then:
>> +
>> +resultant(f, g, x) = (-1)**(Df*Dg) * resultant(g, f, x).
>
> How would if affect this PR if this sign were added to SymPy code?
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/sympy/sympy/pull/10673/files#r53906695